.github/workflows/e2e-test.yaml

```diff
@@ -0,0 +1,28 @@
+name: Run CCNP E2E test
+on:
+   schedule:
+    - cron: '20 10 * * *'
+   workflow_dispatch:
+jobs:
+  E2E_test:
+    name: Publish ccnp package
+    runs-on: tdvm-ut
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Run Test
+        run: |
+           cd ./test/e2e-test/
+           sudo ./ci-test.sh
+      - name: Clean up the environment
+        run: |
+            kind delete cluster --name "my-cluster"
```

Comment:
The best practice in github action workflow is calling single command or script to do various of stuffs in the "run" section.
Since in "Run Test" step, we have used a single shell ci-test.sh, why not use a single shell script in this "Clean up" step?

Reply:
Fixed, add "ci-clean.sh" script.